app/src/main/java/com/example/plantsapp/presentation/ui/notification/broadcastreceiver/NotificationBroadcastReceiver.kt

```diff
@@ -104,10 +114,20 @@ class NotificationBroadcastReceiver : HiltBroadcastReceiver() {
                 putExtra(EXTRA_NOTIFICATION_ID, notificationId)
             }
 
+        private var Intent.plantName: Plant.Name
+            get() = Plant.Name(
+                getStringExtra(EXTRA_PLANT_NAME)
+                ?: error("Plant Name Was Not Passed!")
+            )
+            set(plantName) {
+                putExtra(EXTRA_PLANT_NAME, plantName.value)
+            }
+
```

Comment:
If you use this sowewhere else , how about Create extension file in presentation layer for this thing 
If not - just leave as it

Reply:
used only here